h/templates/accounts/account.html.jinja2

```diff
@@ -5,20 +5,25 @@
 
 {% block page_content %}
   <div class="form-vertical">
+    {% if not feature('activity_pages') -%}
     <h2 class="form-heading">
       <span>{% trans %}Change your email address{% endtrans %}</span>
     </h2>
     <legend>
       {% trans %}Your current email address is:{% endtrans %}
       <strong>{{ email }}</strong>.
     </legend>
-    {{ email_form.render() }}
+    {%- endif %}
+    {{ email_form }}
 
+    {% if not feature('activity_pages') -%}
     <h2 class="form-heading">
       <span>{% trans %}Change your password{% endtrans %}</span>
     </h2>
-    {{ password_form.render() }}
+    {%- endif %}
+    {{ password_form }}
 
+    {% if not feature('activity_pages') -%}
```

Comment:
Not showing this matches the mocks but I think we should continue to provide users with information on what to do to delete their account. My guess is that this would be done with some text and a link in the footer of the form (see the `/signup` page for an example of what that looks like). Thoughts @conordelahunty ?